<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelDialog</class>
 <widget class="QDialog" name="LabelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="projectgroupBox">
     <property name="title">
      <string>Project List</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QListView" name="projectView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>47</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="labelgroupBox">
     <property name="title">
      <string>Label List</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="addButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="5">
       <widget class="QListView" name="labelView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="importlistButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>85</horstretch>
          <verstretch>24</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Import List</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="importtabButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Import Table</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="showButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="removeButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>14</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>importlistButton</zorder>
     <zorder>importtabButton</zorder>
     <zorder>showButton</zorder>
     <zorder>removeButton</zorder>
     <zorder>labelView</zorder>
     <zorder>addButton</zorder>
    </widget>
   </item>
  </layout>
  <zorder>labelgroupBox</zorder>
  <zorder>projectgroupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
